runtime/cmake/platform/toolcfg/openxl.cmake

```diff
@@ -0,0 +1,35 @@
+################################################################################
+# Copyright IBM Corp. and others 2024
+#
+# This program and the accompanying materials are made available under
+# the terms of the Eclipse Public License 2.0 which accompanies this
+# distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+# or the Apache License, Version 2.0 which accompanies this distribution and
+# is available at https://www.apache.org/licenses/LICENSE-2.0.
+#
+# This Source Code may also be made available under the following
+# Secondary Licenses when the conditions for such availability set
+# forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+# General Public License, version 2 with the GNU Classpath
+# Exception [1] and GNU General Public License, version 2 with the
+# OpenJDK Assembly Exception [2].
+#
+# [1] https://www.gnu.org/software/classpath/license.html
+# [2] https://openjdk.org/legal/assembly-exception.html
+#
+# SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0-only WITH Classpath-exception-2.0 OR GPL-2.0-only WITH OpenJDK-assembly-exception-1.0
+################################################################################
+
+list(APPEND OMR_PLATFORM_COMPILE_OPTIONS -O3)
+
+list(APPEND OMR_PLATFORM_CXX_COMPILE_OPTIONS -fno-rtti)
+
+list(APPEND OMR_PLATFORM_COMPILE_OPTIONS -fstack-protector)
+
+if(NOT OMR_OS_ZOS)
+	list(APPEND OMR_PLATFORM_CXX_COMPILE_OPTIONS -qsuppress=1540-1087:1540-1088:1540-1090)
+endif()
+
+# OMR_PLATFORM_CXX_COMPILE_OPTIONS gets applied to the jit (which needs exceptions),
+# so we put these in the CMAKE_CXX_FLAGS instead
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fignore-exceptions")
```

Comment:
This doesn't seem right. JIT uses (throws and catches) exceptions: how does ignoring them make sense?

Reply:
I was trying to recall why I put this here, I think I have found it.
It is the equivalence for this here in xlc.cmake: https://github.com/eclipse-openj9/openj9/blob/2a2df9f1117f0e3dad2ab9022dc6d05734c0cf83/runtime/cmake/platform/toolcfg/xlc.cmake#L35

Mapping of that option goes to `-fignore-exceptions` as per https://www.ibm.com/docs/en/open-xl-c-cpp-zos/2.1?topic=options-mapping

Reply:
I misread the reference in the comment to JIT code: this option is only for non-JIT code.